components/console/helpers/questionhelper.rst

```diff
@@ -289,26 +289,28 @@ from the command line, you need to set the helper input stream::
         // ...
         $commandTester = new CommandTester($command);
 
-        $helper = $command->getHelper('question');
-        $helper->setInputStream($this->getInputStream("Test\n"));
+        $commandTester->setInputs(array('Test'));
         // Equals to a user inputting "Test" and hitting ENTER
-        // If you need to enter a confirmation, "yes\n" will work
+
+        $commandTester->setInputs('This', 'That');
```

Comment:
should be `array('This', 'That')` (same on line 299).

Also, I think we should add the descriptive comment before the line of code.